Open5GS daemon v2.4.4

08/31 10:59:01.495: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 10:59:01.495: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
08/31 10:59:01.502: [sock] ERROR: socket bind(2) [192.168.0.113]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-socket.c:109)
08/31 10:59:01.502: [sock] ERROR: udp_server() [192.168.0.113]:8805 failed (99:Cannot assign requested address) (../lib/core/ogs-udp.c:62)
08/31 10:59:01.502: [app] ERROR: Failed to intialize UPF (../src/upf/app.c:28)
08/31 10:59:01.502: [app] FATAL: Open5GS initialization failed. Aborted (../src/main.c:219)
Open5GS daemon v2.4.4

08/31 11:05:20.369: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 11:05:20.369: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
08/31 11:05:20.378: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
08/31 11:05:20.378: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
08/31 11:05:20.378: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 11:05:20.412: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 11:05:20.413: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 12:32:22.633: [pfcp] ERROR: ogs_pfcp_sendto() failed (101:Network is unreachable) (../lib/pfcp/path.c:115)
08/31 12:32:22.633: [pfcp] ERROR: ogs_pfcp_xact_commit: Expectation `rv == OGS_OK' failed. (../lib/pfcp/xact.c:573)
08/31 12:32:22.914: [app] INFO: SIGHUP received (../src/main.c:58)
Open5GS daemon v2.4.4

08/31 12:32:51.214: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 12:32:51.214: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
08/31 12:32:51.238: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
08/31 12:32:51.238: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
08/31 12:32:51.239: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 12:32:51.334: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 12:32:51.334: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.4

08/31 14:54:05.606: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 14:54:05.606: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
08/31 14:54:05.643: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
08/31 14:54:05.643: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
08/31 14:54:05.644: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 14:54:08.119: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 14:54:08.120: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 14:54:17.970: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
08/31 14:54:17.970: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
08/31 14:54:17.970: [upf] INFO: UE F-SEID[CP:0x1 UP:0x2] APN[mec] PDN-Type[1] IPv4[10.46.0.2] IPv6[] (../src/upf/context.c:361)
08/31 14:54:18.434: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

08/31 17:08:38.652: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
08/31 17:08:38.652: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
08/31 17:08:38.676: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
08/31 17:08:38.676: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
08/31 17:08:38.677: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
08/31 17:08:38.771: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
08/31 17:08:38.771: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
08/31 17:08:50.867: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
08/31 17:08:50.867: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
08/31 17:08:50.868: [upf] INFO: UE F-SEID[CP:0x1 UP:0x2] APN[mec] PDN-Type[1] IPv4[10.46.0.2] IPv6[] (../src/upf/context.c:361)
08/31 17:08:51.322: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

09/04 08:29:53.797: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/04 08:29:53.797: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
09/04 08:29:53.807: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
09/04 08:29:53.807: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
09/04 08:29:53.807: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/04 08:37:48.277: [app] INFO: SIGHUP received (../src/main.c:58)
Open5GS daemon v2.4.4

09/18 09:20:51.391: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:20:51.391: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
09/18 09:20:51.400: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
09/18 09:20:51.400: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
09/18 09:20:51.400: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:20:52.739: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:20:52.740: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
Open5GS daemon v2.4.4

09/18 09:26:03.645: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:26:03.645: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
09/18 09:26:03.656: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
09/18 09:26:03.656: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
09/18 09:26:03.656: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:26:03.814: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:26:03.814: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/18 09:26:15.851: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/18 09:26:15.851: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/18 09:26:15.852: [upf] INFO: UE F-SEID[CP:0x1 UP:0x2] APN[mec] PDN-Type[1] IPv4[10.46.0.2] IPv6[] (../src/upf/context.c:361)
09/18 09:26:16.296: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
Open5GS daemon v2.4.4

09/18 09:41:08.218: [app] INFO: Configuration: '/open5gs/install/etc/open5gs/upf.yaml' (../lib/app/ogs-init.c:129)
09/18 09:41:08.218: [app] INFO: File Logging: '/open5gs/install/var/log/open5gs/upf_mec.log' (../lib/app/ogs-init.c:132)
09/18 09:41:08.225: [pfcp] INFO: pfcp_server() [192.168.0.113]:8805 (../lib/pfcp/path.c:30)
09/18 09:41:08.225: [gtp] INFO: gtp_server() [192.168.0.113]:2152 (../lib/gtp/path.c:30)
09/18 09:41:08.225: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
09/18 09:41:08.348: [pfcp] INFO: ogs_pfcp_connect() [192.168.0.111]:8805 (../lib/pfcp/path.c:59)
09/18 09:41:08.348: [upf] INFO: PFCP associated (../src/upf/pfcp-sm.c:173)
09/18 09:41:20.328: [upf] INFO: [Added] Number of UPF-Sessions is now 1 (../src/upf/context.c:160)
09/18 09:41:20.328: [gtp] INFO: gtp_connect() [127.0.0.4]:2152 (../lib/gtp/path.c:58)
09/18 09:41:20.328: [upf] INFO: UE F-SEID[CP:0x1 UP:0x2] APN[mec] PDN-Type[1] IPv4[10.46.0.2] IPv6[] (../src/upf/context.c:361)
09/18 09:41:20.809: [gtp] INFO: gtp_connect() [192.168.0.131]:2152 (../lib/gtp/path.c:58)
09/18 10:12:35.631: [app] INFO: SIGHUP received (../src/main.c:58)
